Cargo.toml

```diff
@@ -28,6 +28,7 @@ slog = "2.4.1"
 slog-async = "2.3.0"
 slog-term = "2.4.0"
 typemap = "0.3.3"
+serde_urlencoded = "0.5.5"
```

Comment:
Could we use `serde_qs`? Maybe more powerful than nescessary, but we already depend on it for forms.

Reply:
See my response to this question [here](https://github.com/rustasync/tide/pull/175#issuecomment-484542425).

Reply:
IMO, it could make sense to move away from `serde_qs` for form handling, too, unless we can get a maintainer to help cover issues for that library. 

This isn't to say I don't want to use `serde_qs` -- I understand why the library exists, but I don't want to end up depending on a library that has fixes ready to go but are unapplied.